scipy/optimize/_shgo.py

```diff
@@ -1,24 +1,28 @@
-"""
-shgo: The simplicial homology global optimisation algorithm
-"""
-
-import numpy as np
+"""shgo: The simplicial homology global optimisation algorithm."""
+from collections import namedtuple
 import time
 import logging
 import warnings
+import sys
+
+import numpy as np
+
 from scipy import spatial
 from scipy.optimize import OptimizeResult, minimize, Bounds
+from scipy.optimize._optimize import MemoizeJac
 from scipy.optimize._constraints import new_bounds_to_old
-from ._optimize import _wrap_scalar_function
-from scipy.optimize._shgo_lib.triangulation import Complex
+from scipy._lib._util import _FunctionWrapper
 
+from scipy.optimize._shgo_lib._complex import Complex
 
-__all__ = ['shgo']
+__all__ = ['shgo', 'SHGO']
```

Comment:
```suggestion
__all__ = ['shgo']
```

Not 100% sure but this hopefully resolves the last refguide error.

![image](https://user-images.githubusercontent.com/40656107/218270587-a9f38c9d-571e-4916-bf3e-073eb46cbf32.png)